e2e_tests/provider_cfg/cryptoauthlib/config_608a.toml

```diff
@@ -0,0 +1,33 @@
+[core_settings]
```

Comment:
Do we use that file on the CI?

Reply:
No - it is only for our convenience. Hope it is not a big issue.

Reply:
Ah as in you are using it for local testing? It's fine for me but I think it would be nice to put this information as comment in the beginning of the file if you decide to keep it.

Reply:
I will add such comments (there is another similar file).

Reply:
Added.